extensions/cornerstone-dicom-sr/src/utils/hydrateStructuredReport.js

```diff
@@ -166,17 +193,26 @@ export default function hydrateStructuredReport(
         },
       };
 
-      const source = MeasurementService.getSource(
+      const source = measurementService.getSource(
         CORNERSTONE_3D_TOOLS_SOURCE_NAME,
         CORNERSTONE_3D_TOOLS_SOURCE_VERSION
       );
       annotation.data.label = getLabelFromDCMJSImportedToolData(toolData);
+      annotation.data.finding = convertCode(
+        codingValues,
+        toolData.finding?.[0]
+      );
+      annotation.data.findingSites = convertSites(
+        codingValues,
+        toolData.findingSites
+      );
+      annotation.data.site = annotation.data.findingSites?.[0];
 
       const matchingMapping = mappings.find(
         m => m.annotationType === annotationType
       );
 
-      MeasurementService.addRawMeasurement(
+      measurementService.addRawMeasurement(
```

Comment:
I don't like how each PR will have one or two change of letter casing. Those PRs should be separate and a complete rename of one service like what we did for Cine and Toolbar. Please take these out